### PR TITLE
Creating accessibility

### PR DESCRIPTION
**Pull request by annaeliana7**
In this branch, we added accessibility to the project. We did the following changes:

- Image text alternatives → We modified the alt attribute to image dropdown-arrow to make it more descriptive.
  https://github.com/annaeliana7/Portfolio/blob/ea0e9dc63d6b55af8f4fb573e7ea46eac6f91ac1/index.html#L188

- Page titles → We changed the Page title to `Anna | Portfolio`.
https://github.com/annaeliana7/Portfolio/blob/d84da3909ff9e7032ae716aa1ddd5d2467ec32b9/index.html#L8

Check accessibility: 

- Text headings - no issues found
- Color contrast - no issues found
- Resize - No issues found
- Interaction - No issues found
- Moving content - No issues found
- Multimedia - No issues found
- The basic structure of the page - No issues were found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -185,7 +185,7 @@ <h1>About Myself</h1>
                 <li>
```
Comment (HAMMAS-SALEEM) [inline index.html:185]: My Logo
get-in-touch
html
Navlink
see-project-button
technology

- [ ] You can see in the above images that some of your things in your webpage doesn't have a color contrast of 4.5:1. Kindly make it to make your application accessible for all types of users specially color blind users 😊

File: index.html [hunk 2]
```diff
@@ -185,7 +185,7 @@ <h1>About Myself</h1>
...
                     <div class="title border-none">
                         <h5>Languages</h5>
-                        <img src="images/Dropdown-down arrow.svg" alt="dropdown-arrow">
+                        <img src="images/Dropdown-down arrow.svg" alt="dropdown-arrow icon">
                     </div>
```
Comment (HAMMAS-SALEEM) [inline index.html:189]: chanes

- [x] You have changed this line but have not added the link of this file to your pull request summary which is a requirement as you can see in the image above ⬆️

Hint: You can see this [pull request summary](https://github.com/orozCoding/myportfolio/pull/8) to get an idea what we are talking about 😊

File: index.html (hunk 1 above)
Comment (HAMMAS-SALEEM) [inline index.html:185]: page-title

- [ ] Your page title should be strong and good in this case your page title is just Portfolio 🤔 Kindly change it to `Anna | Portfolio` to make it good and strong 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @annaeliana7 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github Flow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrantoshB** [comment]
Hi @HAMMAS-SALEEM, I appreciate that you've reviewed this project of my partner. But I mistakenly submitted her PR link from my dashboard and you've reviewed it. But I want you to please mark my submitted link as invalid so that I can submit my PR link. 

Thanks.  Have a good day 😇

File: index.html (hunk 1 above)
Comment (annaeliana7) [reply index.html:185]: @HAMMAS-SALEEM, thank you for your comment. If you check the figma design, I made My Logo larger than the figma design. I also followed what I was asked to do with regards to using figma's choice of colors.

Comment (Santiago220991) [reply index.html:185]: As the last reviewer suggested kindly fix the color contrast ratio of your buttons. Feel free to change the color and the size text to fulfill this requirement. :pray: :pray: You can use[ this tool ](https://www.aditus.io/button-contrast-checker/)to ensure it.

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @annaeliana7 @PrantoshB, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: Applied most of the changes suggested by the last reviewer.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**